src/components/data-exploration/generic-gene-table/CreateCellSetModal.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import {
+  Modal, Form, Button, Select, InputNumber, Row, Alert, Col,
+} from 'antd';
+import PropTypes from 'prop-types';
+
+const formItemStyle = { margin: '0.375rem 0' };
+
+const CreateCellSetModal = (props) => {
+  const { selectedGenes, onCancel } = props;
+  const [form] = Form.useForm();
+  const intialFormValues = selectedGenes.map((gene) => ({ selectedGenes: gene }));
+
+  const comparisonOptions = [{
+    value: 'greaterThan',
+    label: 'Greater than',
+  }, {
+    value: 'lessThan',
+    label: 'Less than',
+  },
+  ];
+
+  return (
+    <Modal
+      visible
+      title='Create a new cell set based on gene expression'
+      onCancel={onCancel}
+      footer={[<Button type='primary'>Create</Button>]}
```

Comment:
Nothing happens when I click "Create", is this intentional? Are we planning to implement the functionality of create after the backend is merged?

Reply:
Yes